src/main.rs

```diff
@@ -70,6 +92,116 @@ impl BFInterp {
             MetadataOperand => todo!(),
         }
     }
+
+    fn load(&mut self, op: &Operand, dest: &name::Name) {
```

Comment:
I think `dest` should be called `src` here, right, as that's where we load from?

Reply:
I think dest is where we load to rather than from so I use dest as the identifier.

Reply:
Oh yes. My bad!